src/utils/constant/form.js

```diff
@@ -0,0 +1 @@
+export const ENTER = "Enter";
```

Comment:
consider adding a context to the constant name.

```suggestion
export const KEY_ENTER = "Enter";
```